File: c#/classTest.cs
using System;

class Person{
	private string name;
	private int age;

	public Person(string name, int age){
		this.name = name;
		this.age = age;
	}

	public void say(){
		Console.WriteLine("{0} {1}", this.name, this.age);
	}
}

class Test{
	static void Main(string[] args){
		Person a = new Person("Stanley", 20);
		a.say();
	}
}

File: lua/classTest.lua

Person = {}

function Person:new(name, age)
	self.name = name
	self.age = age
	return self
end

function Person:say()
	print(self.name, self.age)
end

function test()
	a = Person:new("Stanley", 20)
	a:say()
end
